contracts/core/extensions/CoreIssuanceOrder.sol

```diff
@@ -152,7 +152,7 @@ contract CoreIssuanceOrder is
         // Settle Order
         settleOrder(order, _fillQuantity, _orderData);
 
-        //Issue Set
+        //Issue Settle
```

Comment:
need a space after `//`, i think the precommit linting should catch these things now though?